### PR TITLE
chore: update profile dropdown

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/rnd-dev-team#1875

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

From https://deploy-preview-2304--regen-marketplace.netlify.app/
1. Login and check the new profile dropdown UI
2. Log in with Keplr then switch account from Keplr. You should see a warning modal in the app and the Keplr "prove ownership" popup
   1. If you approve then you're logged out from the previous account and logged in with the new one
   2. If you reject, then you'll still be logged in with the previous account but the wallet address won't be the same as the one from your current account so as soon as you want to perform an on chain action, you'll get a modal saying to switch to the right account in Keplr
   3. If you switch back to the initial account from Keplr, then you are still logged in with the initial account

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie see testing instructions

**Comment by erikalogie** [comment]
@blushi this looks great! 

Only a few small things:
- Can we move the icon down just a bit so that it is vertically centered with the text? https://monosnap.com/file/HYa6bqGnq0vymjqegx1iDR7FuT3zdd
- Can we replace all the icons with the gradient version? I just think it looks a little bit nicer. It's fine, even preferable if they are the same gradient icons on the actual tabs on the profile page itself as well. I added all the gradient icons to the attached file, not sure if that's helpful.

[Icons2.zip](https://github.com/regen-network/regen-web/files/14449286/Icons2.zip)

**Comment by erikalogie** [comment]
@blushi also can you move the dropdown over just a bit to the right so it totally right aligns with the user info box abox? https://monosnap.com/file/2q2ZbdKMNSoTtyBOWNVQait6kTTAMc

File: web-components/src/components/header/components/NavLink.tsx [hunk 1]
```diff
@@ -33,18 +34,23 @@ export type NavLinkProps = {
...
 export const NavLink: React.FC<React.PropsWithChildren<NavLinkProps>> = ({
   children,
   href,
   pathname,
   overrideClassname,
+  className,
 }) => {
   const { classes: styles } = useNavLinkStyles({ isActive: pathname === href });
-
+  console.log(className);
```
Comment (aaronc) [inline web-components/src/components/header/components/NavLink.tsx:48]:
```suggestion
```

**Review by aaronc — APPROVED** [review]
utACK pending @erikalogie's approval and console.log cleanup

**Comment by blushi** [comment]
@erikalogie I've addressed your review comments.

**Comment by erikalogie** [comment]
@blushi this looks awesome! Nice work!

**Pull request merged**